applications/osb-portal/src/pages/RepositoriesPage/RepositoriesPage.tsx

```diff
@@ -289,7 +289,15 @@ export const RepositoriesPage = ({ user }: { user: UserInfo }) => {
 
         {repositories ?
           <Box className="verticalFill" style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-between' }}>
-            <Repositories repositories={repositories} handleRepositoryClick={(repositoryId: number) => openRepoUrl(repositoryId)} refreshRepositories={() => updateList(tabValue)} />
+            <Repositories
+            repositories={repositories}
+            refreshRepositories={() => updateList(tabValue)}
+            searchFilterValues={searchFilterValues}
+            handleRepositoryClick={(repositoryId: number) => openRepoUrl(repositoryId)}
+            handleTagClick={ (tag: Tag) => searchFilterValues.tags.indexOf(tag.tag) === -1 ? setSearchFilterValues({ ...searchFilterValues, tags: searchFilterValues.tags.concat(tag.tag)}) : console.log("Tag exists, no op.") }
```

Comment:
Why the console.log? Would rather do nothing, or a console.debug

Reply:
Yeh, i wasn't sure of "how to do nothing". `pass` doesn't work, because it's an expression. Should I just use `null`?

Reply:
Used `null` now, seems to work fine.